examples/hitl/rearrange_v2/ui.py

```diff
@@ -346,6 +580,19 @@ def _place_object(self) -> None:
         point = self._place_selection.point
         normal = self._place_selection.normal
         receptacle_object_id = self._place_selection.object_id
+
+        # check for a valid Receptacle mapping for the place point
+        # TODO: cache this ground truth mapping in the trajectory?
+        (
+            _placement_receptacle,
+            _confidence,
+            _info_text,
+        ) = self.get_place_obj_receptacle_and_confidence(
+            point, receptacle_object_id
+        )
+        print(
+            f"Placed object on Receptacle '{_placement_receptacle}', confidence[0,1]={_confidence}. Info text: {_info_text}"
+        )
```

Comment:
Replace this with necessary event logic to cache the new object->Receptacle parenting